apps/autoscheduler/AutoSchedule.cpp

```diff
@@ -3098,11 +3110,11 @@ IntrusivePtr<State> optimal_schedule_pass(FunctionDAG &dag,
             // The user has set HL_CYOS, and wants to navigate the
             // search space manually.  Discard everything in the queue
             // except for the user-chosen option.
-            aslog(0) << "\n--------------------\n";
```

Comment:
This is asking for human input, and should always be printed

Reply:
I still shudder at the "ask for human input" part here :-)

Reply:
It's a debugging tool for people working on the autoscheduler